# Image processing settings
std_msgs/Int32 tomato_radius_min
std_msgs/Int32 tomato_radius_max
std_msgs/Int32 tomato_distance_min
std_msgs/Int32 dp
std_msgs/Int32 param1
std_msgs/Int32 param2
